estuary/ui
# Correct the function of the collections table materialize CTA

## Issues

The issues directly below are completely resolved by this PR:
https://github.com/estuary/ui/issues/892

## Changes

### 892

The following features are included in this PR:

* Update how the value of a selected collections table row is determined.

* Removed the explicit definition of a click handler for a capture table row.

## Tests

### Manually tested

Approaches to testing are as follows:

* Select a single row of the collections table on the _Collections_ page, click the _Materialize_ CTA, enter the materialization create workflow, and observe that the selected collection appears in the _Source Collections_ section of the form.

* Select multiple rows of the collections table on the _Collections_ page, click the _Materialize_ CTA, enter the materialization create workflow, and observe that all the selected collections appear in the _Source Collections_ section of the form.

* Select a single row of the collections table on the _Collections_ page, click the _Materialize_ CTA, select a connector for which a materialization exists, select an existing materialization on the existing entity check page, enter the materialization edit workflow, and observe that the selected collection appears in the _Source Collections_ section of the form and is bound to the drafted specification.

* Select multiple rows of the collections table on the _Collections_ page, click the _Materialize_ CTA, select a connector for which a materialization exists, select an existing materialization on the existing entity check page, enter the materialization edit workflow, and observe that all the selected collections appear in the _Source Collections_ section of the form and are bound to the drafted specification.

* Select a single row of the captures table on the _Captures_ page, click the _Materialize_ CTA, enter the materialization create workflow, and observe that the collections bound to the selected capture appear in the _Source Collections_ section of the form.

* Select multiple rows of the captures table on the _Captures_ page, click the _Materialize_ CTA, enter the materialization create workflow, and observe that the collections bound to the selected capture appear in the _Source Collections_ section of the form.

### Automated tests

-   None

## Screenshots

N/A
